cnf-tests/pod-utils/hwlatdetect-runner/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"flag"
+	"k8s.io/klog"
+	"os/exec"
+	"strconv"
+	"time"
+
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/pod-utils/pkg/node"
+)
+
+const hwlatdetectBinary = "/usr/bin/hwlatdetect"
+const cmdExecuter = "/usr/bin/python3"
+
+func main() {
+	klog.InitFlags(nil)
+
+	threshold := flag.Int("threshold", 2, " value above which is considered an hardware latency")
+	hardlimit := flag.Int("hardlimit", 2, " value above which the test is considered to fail")
+	duration := flag.Duration("duration", time.Second * 15, "total time to test for hardware latency: <n>{smdw}")
+	window := flag.Duration("window", time.Second, "time between samples")
+	width := flag.Duration("width", time.Microsecond * 950000, "time to actually measure")
+	hwlatdetectStartDelay := flag.Duration("hwlatdetect-start-delay", 0, "delay in second before running the hwlatdetect binary")
+
+	flag.Parse()
+
+	err := node.PrintInformation()
+	if err != nil {
+		klog.Fatalf("failed to print node information: %v", err)
+	}
+
+	time.Sleep(*hwlatdetectStartDelay * time.Second)
+
+	hwlatdetectArgs := []string{
+		hwlatdetectBinary,
+		"--threshold", strconv.Itoa(*threshold),
+		"--hardlimit", strconv.Itoa(*hardlimit),
+		"--duration", duration.String(),
+		"--window", window.String(),
```

Comment:
This is a duration, if you want to fetch the seconds you need to use window.Seconds() otherwise it will be rendered like `52s`

Reply:
From what I see, the tool only knows how to deal with only one measurement unit, meaning:
```5s``` is legal but ```5m5s``` and ```5``` isn't.
So not sure Duration is a good choice here because 5 minutes in duration will be translated to ```5m0s``` and the tool doesn't know how to deal with that, I'll try to translate everything into microsecond which is the most common measurement unit in this context